### PR TITLE
enable profiles

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/145

### CODE REVIEW

File: octobot/cli.py [hunk 1]
```diff
@@ -112,20 +112,29 @@ def start_octobot(args):
         logger.info("Loading config files...")
 
         # configuration loading
-        config = common_config.load_config(error=False, fill_missing_fields=True)
-
-        if config is None and common_config.is_config_empty_or_missing():
-            logger.info("No configuration found creating default...")
+        config_path = configuration.get_user_config()
```
Comment (Herklos) [inline octobot/cli.py:115]: :+1:

File: octobot/configuration_manager.py [hunk 2]
```diff
@@ -118,3 +126,35 @@ def init_config(
...
+    shutil.copyfile(constants.DEFAULT_PROFILE_AVATAR,
+                    os.path.join(profile_folder, constants.DEFAULT_PROFILE_AVATAR_FILE_NAME))
+    if os.path.isdir(tentacles_manager_constants.USER_REFERENCE_TENTACLE_CONFIG_PATH):
+        shutil.copyfile(tentacles_manager_constants.USER_REFERENCE_TENTACLE_CONFIG_FILE_PATH,
+                        os.path.join(profile_folder, tentacles_manager_constants.constants.CONFIG_TENTACLES_FILE))
+        shutil.copytree(tentacles_manager_constants.USER_REFERENCE_TENTACLE_SPECIFIC_CONFIG_PATH,
+                        os.path.join(profile_folder, tentacles_manager_constants.TENTACLES_SPECIFIC_CONFIG_FOLDER))
+
+
+def migrate_from_previous_config(config):
```
Comment (Herklos) [inline octobot/configuration_manager.py:147]: :+1:

File: octobot/configuration_manager.py [hunk 3]
```diff
@@ -118,3 +126,35 @@ def init_config(
...
+    if common_constants.CONFIG_CRYPTO_CURRENCIES in config.config:
+        # config migration required
+        # add missing exchange enabled config
+        for exchange_config in config.config[common_constants.CONFIG_EXCHANGES].values():
+            exchange_config[common_constants.CONFIG_ENABLED_OPTION] = \
+                exchange_config.get(common_constants.CONFIG_ENABLED_OPTION, True)
+        for key in ("tentacles-packages", "performance-analyser", "PERF", "SAVE_EVALUATIONS"):
```
Comment (Herklos) [inline octobot/configuration_manager.py:154]: Shouldn't it be a constant in constants ? To be more accessible

Comment (GuillaumeDSM) [reply octobot/configuration_manager.py:154]: I removed the from constants to that we know that we don't use them anymore, I think it's better to leave it this way and just remove them when we will need to updater the configuration migrator in later versions

Comment (Herklos) [reply octobot/configuration_manager.py:154]: ok

**Review by Herklos — APPROVED** [review]

**Pull request merged**